modules/fuzztest/20240813.0/patches/update_module_bazel_version.patch

```diff
@@ -0,0 +1,26 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -1,6 +1,6 @@
+ module(
+     name = "fuzztest",
+-    version = "head",
++    version = "20240813.0",
+     repo_name = "com_google_fuzztest",
+ )
+ 
```

Comment:
Same here.

Reply:
Argh. I'll clean up and generate a proper patch file again directly without manually copying the patch suggestion.